### PR TITLE
Update title in tribute.html

### PR DESCRIPTION
**Pull request by enocol**
This project is to replicate tributes. view the page and make comments about project.

### CODE REVIEW

File: tribute.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <title>Document1</title>
```
Comment (enocol) [inline tribute.html:8]: You are doing great so far. Please permit me to mention here that, your document is missing a proper document title. I believe it is always good practice to give your document a descriptive title  to tell your visitors what sort of information may be found on your website,

File: tribute.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
 </head>
 <body>
     <div class="container">
```
Comment (enocol) [inline tribute.html:11]: Please also permit me to mention that, your document is good but follows no semantics. It's always recommended to use semantics tags in your HTML. You can use semantics tags like <header> to group related elements like the <nav> or a logo that appear on the head of your document. As a recommendation, please use semantic tags.

**Review by enocol — COMMENTED** [review]
May I say that, a website title is a very important entity to your website Please always consider this when creating an HTML document so that you give your users an initial idea about the content of your website.  Please also kindly consider using semantic tags in your document as this helps search engines and other user devices determine the importance and content of your web pages.

**Review by Gambit142 — COMMENTED** [review]
Hi @enocol 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Good use of inline comments ✔️ 
- The comments were polite ✔️ 
- The comments were clear and easy to understand ✔️
- The comments explained the reasons for requesting changes ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**